Artifacts/PgSql/Structure/Admin/0160-Rename-OdsInstanceContext.sql

```diff
@@ -0,0 +1,50 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE dbo.OdsInstanceContext
+	DROP CONSTRAINT IF EXISTS FK_OdsInstanceContext_OdsInstanceId_OdsInstanceId;
+
+ALTER TABLE dbo.OdsInstanceContext
+	DROP CONSTRAINT IF EXISTS UC_OdsInstanceDerivative_OdsInstanceId_ContextKey;
+
+ALTER TABLE dbo.OdsInstanceContext
```

Comment:
IF EXISTS (SELECT column_name
               FROM information_schema.columns
               WHERE table_name = 'OdsInstanceContext'
                 AND column_name = 'OdsInstanceId') THEN
        ALTER TABLE dbo.OdsInstanceContext
        RENAME COLUMN OdsInstanceId TO OdsInstance_OdsInstanceId;
    END IF;